packages/gotrue/test/client_test.dart

```diff
@@ -43,9 +43,10 @@ void main() {
           'apikey': anonToken,
         },
         asyncStorage: asyncStorage,
+        flowType: AuthFlowType.implicit,
```

Comment:
I'm unsure about this change as it changes the whole file to now use implicit again, doesn't it? We should properly do all tests for pkce and implicit.

Reply:
>I'm unsure about this change as it changes the whole file to now use implicit again, doesn't it?

Yes, it does. The issue here is that implicit flow was not tested anywhere and the `PKCE enabled client` group doesn't add anything. Now we have the same set of tests that we always had before v2.

>We should properly do all tests for pkce and implicit.

At least now we have the same set of test cases as the js SDK. We can always add more tests if we want to in the future.